Sources/CashUI/UI/OtherUI/MapViewController.swift

```diff
@@ -40,7 +40,7 @@ class MapViewController: UIViewController {
         var _locationManager = CLLocationManager()
         _locationManager.delegate = self
         _locationManager.desiredAccuracy = kCLLocationAccuracyBest
-        
+
```

Comment:
Looking at these changes, it is and will be unclear to me (in the future) to know where the changes to force light colored maps happened.
We should avoid mixing formatting issues with other changes, as we may loose the real purpose of the change.
If you want to have a change in the formatting, we should do those in a separate PR.